Formula/libglib.rb

```diff
@@ -0,0 +1,118 @@
+class Libglib < Formula
+  desc "Core application library for C"
+  homepage "https://developer.gnome.org/glib/"
+  url "https://download.gnome.org/sources/glib/2.72/glib-2.72.1.tar.xz"
+  sha256 "c07e57147b254cef92ce80a0378dc0c02a4358e7de4702e9f403069781095fe2"
+  license "LGPL-2.1-or-later"
+
+  livecheck do
+    formula "glib"
+  end
+
+  keg_only "both libglib and glib provide headers and libraries"
```

Comment:
Do they provide the same ones? Can we remove them from glib then?

Reply:
> Can we remove them from glib then?

Yes, we can. But we need to migrate downstream formulae to depend on `libglib` first.

Reply:
Since they will already depend libglib by extension after this PR  I would assume those files can already be removed from glib now.

Reply:
> I would assume those files can already be removed from glib now.

https://github.com/Homebrew/homebrew-core/blob/c936db0908813c0a6b42cf9c0a819a3ecee3047d/Formula/cairo.rb#L87-L104

Many formulae references `glib` by `Formula["glib"].opt_{include,lib}` rather than the files in `$HOMEBREW_PREFIX/{include,lib}`.

Reply:
I don't think the approach here works, since there are executables in `glib` that will require libraries in `libglib`. However, there are no changes in `glib` that points those executables to `libglib`.

Not to mention that these two formulae are essentially building the same things twice.

Reply:
I agree that we should not ship the same thing twice in two formulae. However, I think we should do that later in another PR rather than here. Many formulae use the libraries and headers in `glib`. Should we migrate then in this PR too?

Reply:
Fixing these formulae so that the same things aren't shipped twice should be fixed here first. The dependents can be migrated separately when that's done.